packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -398,6 +454,19 @@ class ChartPainter extends CustomPainter {
     }
   }
 
+  void drawSelection(Canvas canvas, double x, y) {
+    final paint = Paint()
+      ..strokeWidth = 2.0
+      ..color = colorScheme.hoverSelectionBarColor;
+
+    // Draw the yellow vertical selection bar.
```

Comment:
nit: remove comment that this is yellow.

Reply:
Done.